source/_docs/filezilla.md

```diff
@@ -77,6 +77,17 @@ Error:            Could not connect to server
 Double check settings and resolve typos to fix this issue.
 
 ### Site Manager
-Features offered in the Filezilla Site Manager (like [Synchronized Browsing](https://wiki.filezilla-project.org/Using#Synchronized_Browsing){.external}) are not supported because the Pantheon platform sometimes migrates sites across appservers without warning and the non-static binding string will change. This means that while you can set up your site in the Site Manager, you will need to reconfigure the login information and file paths whenever the dev environment site binding changes.
+Features offered in the FileZilla Site Manager (like [Synchronized Browsing](https://wiki.filezilla-project.org/Using#Synchronized_Browsing){.external}) are not supported because the Pantheon platform sometimes migrates sites across appservers without warning and the non-static binding string will change. This means that while you can set up your site in the Site Manager, you will need to reconfigure the login information and file paths whenever the dev environment site binding changes.
 
 The value for **Default remote directory** in the Site Manager can be copied from the **Remote site** field in the main window, and you can append `code` to the path to synchronize with your local codebase. Rememebr that the site binding is subject to change. 
+
+### Quickconnect is refusing to connect
+
+The following error is always thrown when using Quickconnect option in FileZilla:
+
+```bash
+Error:        	Cannot establish FTP connection to an SFTP server. Please select proper protocol.
+Error:        	Critical error: Could not connect to server
+```
+
+Quickconnect do not give options for protocol selection. Please see how to properly [Configure FileZilla](docs/filezilla/#configure-filezilla).
```

Comment:
Needs grammar edits, and suggest that we just reference the need to specify `sftp://` here rather than point them to another doc to find that out.